src/openlifu/db/photoscan.py

```diff
@@ -0,0 +1,229 @@
+        else:
+            return json.dumps(self.to_dict(), indent=4, cls=PYFUSEncoder)
+
+    @staticmethod
+    def from_dict(d:Dict):
+        """
+        Create a Photoscan from a dictionary
+        param d: Dictionary of photoscan parameters.
+        returns: Photoscan object
+        """
+        return Photoscan(**d)
+
+    def to_dict(self):
+        """
+        Convert the photoscan to a dictionary
+        returns: Dictionary of photoscan parameters
+        """
+        d = self.__dict__.copy()
+        return d
+
+    @staticmethod
+    def from_file(filename):
+        """
+        Load a Photoscan from a metadata file
+        :param filename: Name of the file
+        """
+        with open(filename) as f:
+            return Photoscan.from_dict(json.load(f))
+
+    def to_file(self, filename):
+        """
+        Save the photoscan to a file.
+        :param filename: Name of the file
+        """
+        Path(filename).parent.parent.mkdir(exist_ok=True) #photoscan directory
+        Path(filename).parent.mkdir(exist_ok=True)
+        with open(filename, 'w') as file:
+            file.write(self.to_json(compact = False))
+
+def load_data_from_filepaths(model_abspath: str, texture_abspath: str) -> Tuple[vtk.vtkPolyData, vtk.vtkImageData]:
+    """
+    This function directly returns the data without creating a photoscan object
+    param model_abspath: absolute filepath to model data
+          texture abspath: absolute filepath to texture data
+    Returns: Photoscan data as (model_vtkpolydata, texture_vtkimagedata)
+    """
+    model_polydata  = load_model(model_abspath)
+    texture_imagedata = load_texture(texture_abspath)
+
+    return (model_polydata, texture_imagedata)
+
+def load_data_from_photoscan(photoscan: Photoscan, parent_dir) -> Tuple[vtk.vtkPolyData, vtk.vtkImageData]:
+    """
+    param parent_dir: parent directory containing model and texture data filepaths
+    Returns: Photoscan data as (model_vtkpolydata, texture_vtkimagedata)
+    """
+    model_polydata  = load_model(Path(parent_dir)/photoscan.model_filename)
+    texture_imagedata = load_texture(Path(parent_dir)/photoscan.texture_filename)
+
+    return (model_polydata, texture_imagedata)
+
+def load_model(file_name):
+    """ This function assumes that the model is saved following the LPS coordinate system."""
+    if not Path(file_name).exists():
+        raise FileNotFoundError(f'Model filepath does not exist: {file_name}')
+    mesh = read_as_vtkpolydata(file_name)
+    mesh_ras = convert_between_ras_and_lps(mesh)
+
+    return mesh_ras
+
+def load_texture(file_name):
+    if not Path(file_name).exists():
+        raise FileNotFoundError(f'Texture data filepath does not exist: {file_name}')
+    texture = read_as_vtkimagedata(file_name)
+    return texture
+
+def read_as_vtkpolydata(file_name):
+
+    valid_suffixes = ['.g', '.obj', '.stl', '.ply', '.vtk', '.vtp']
```

Comment:
A more pretty approach could be to replace `valid_suffixes` by 

```
suffix_to_reader_dict = {
    '.ply' : vtk.vtkPLYReader,
    '.vtp' : vtk.vtkXMLPolyDataReader,
    # etc.
}
```

and then you can be like `reader = suffix_to_reader_dict[ext]()` instead having so many `elif`s